airbyte-cdk/java/airbyte-cdk/db-sources/src/main/java/io/airbyte/cdk/integrations/debezium/internals/DebeziumPropertiesManager.java

```diff
@@ -45,10 +45,7 @@ public Properties getDebeziumProperties(
     props.setProperty("max.batch.size", "2048");
     props.setProperty("max.queue.size", "8192");
 
-    // Disabling retries because debezium startup time might exceed our 60-second wait limit
-    // The maximum number of retries on connection errors before failing (-1 = no limit, 0 = disabled, >
-    // 0 = num of retries).
-    props.setProperty("errors.max.retries", "0");
+    props.setProperty("errors.max.retries", "5");
```

Comment:
I get that 5 is totally arbitrary, and is more than 0.  Maybe this should be a constant that can be overridden by ENV if needed?

Reply:
Totally arbitrary. Is there a quick/lightweight way to override this via env?

Ideally wanted to avoid exposing this and other things that are debezium properties elsewhere (whether in the config/env etc)